java/src/test/java/io/cucumber/gherkin/GherkinLineTest.java

```diff
@@ -218,6 +218,24 @@ void startsWithTitleKeyword() {
         assertFalse(gherkinLine.startsWithTitleKeyword("Feat"));
     }
 
+    @Test
+    void startsWithTitleKeyword_short() {
```

Comment:
`_short` and `_mini` could be more descriptive of the condition that is actually being tested.